src/docs/ganache/reference/cli-options.md

```diff
@@ -0,0 +1,970 @@
+new executable transactions are mined instantly.
+
+The defaults is `0`.
+
+The alias `--blockTime` has been deprecated, use `-b` instead.
+
+**`--miner.defaultGasPrice`**, **`g`**
+
+=== "Syntax"
+
+    ```bash
+    --miner.defaultGasPrice=<STRING>
+    ```
+
+=== "Example"
+
+    ```bash
+    --miner.defaultGasPrice="0x87369400"
+    ```
+
+Set the default gas price in WEI for transactions if not specified.
+
+The default is `0x77359400` (2 GWEI).
+
+The alias `--gasPrice` has been deprecated, use `-g` instead.
+
+**`--miner.blockGasLimit`**, **`l`**
+
+=== "Syntax"
+    ```bash
+    --miner.blockGasLimit=<STRING>
+    ```
+
+=== "Example"
+
+    ```bash
+    --miner.blockGasLimit="0x87369400"
+    ```
+
+Set the block gas limit in WEI.
+
+The default is `0x1c9c380` (30 million WEI).
+
+The alias `--gasLimit` has been deprecated, use `-l` instead.
+
+**`--miner.defaultTransactionGasLimit`**
+
+=== "Syntax"
+    ```bash
+    --miner.defaultTransactionGasLimit=<STRING>
+    ```
+
+=== "Example"
+
+    ```bash
+    --miner.defaultTransactionGasLimit="16F30"
+    ```
+
+Set the default transaction gas limit in WEI. Set to `estimate` to use an estimate (slows down transaction execution by 40%+).
+
+The defaults is `0x15f90`.
+
+**`--miner.difficulty`**
+
+=== "Syntax"
+    ```bash
+    --miner.difficulty=<STRING>
+    ```
+
+=== "Example"
+
+    ```bash
+    --miner.difficulty="0x2"
+    ```
+
+Set the block difficulty.
+
+The default is `0x1`.
+
+**`--miner.callGasLimit`**
+
+=== "Syntax"
+    ```bash
+    --miner.callGasLimit=<STRING>
+    ```
+
+=== "Example"
+
+    ```bash
+    --miner.callGasLimit="0x58af080"
+    ```
+
+Set the transaction gas limit in WEI for `eth_call` and `eth_estimateGas` calls.
+
+The default is `0x2faf080`.
+
+**`--miner.instamine`**
+
+=== "Syntax"
+    ```bash
+    --miner.instamine=<STRING>
+    ```
+
+=== "Example"
+
+    ```bash
+    --miner.instamine="strict"
+    ```
+
+Set the `instamine` mode to either `eager` or `strict`. In `eager` mode a transaction is included in a block before its hash
+is returned to the caller. In `strict` mode a transaction's hash is returned to the caller before the transaction is included in a block.
+This value has no effect if `--miner.blockTime` is *not* `0` (the default).
+
+The default is `eager`.
+
+**`--miner.coinbase`**
+
+=== "Syntax"
+    ```bash
+    --miner.coinbase=<STRING>
+    ```
+
+=== "Example"
+
+    ```bash
+    --miner.coinbase="0xfe3b557e8fb62b89f4916b721be55ceb828dbd73"
+    ```
+
+Sets the mining reward address.
+
+The default is `0x0000000000000000000000000000000000000000`.
+
+**`--miner.extraData`**
+
+=== "Syntax"
+    ```bash
+    --miner.extraData=<STRING>
+    ```
+
+=== "Example"
+
+    ```bash
+    --miner.extraData="0x444F4E27542050414E4943202120484F444C2C20484F444C2C20484F444C2021"
+    ```
+
+A hex string representing the 32 bytes included in the extra data field of a mined block.
```

Comment:
Until here, the option descriptions prescribe actions, since they start with "Set to..." / "Set the...". From this option forward, the descriptions omit "set" and only state the information required as arguments. We should use one or the other form for all the descriptions (I prefer the latter).

Reply:
This is what happens when you split work over two days :)

Yes I prefer the latter